### PR TITLE
Init K8s Upstream Marketing Team CHARTER.md

### LINKED ISSUE #4437: mktg team needs a charter
the upstream marketing team needs a small charter. its a new kind of group; will be good to help define it more. want to make sure scope/out of scope is set (boundaries between sig-docs-blog, cncf, etc.)

more notes about charter/purpose/mission in mtg notes
https://docs.google.com/document/d/1IlHAJ131akGhI5ffF4OoVW0PrsVY4C0BB8l-UyQaQVo/edit?usp=sharing

### PR DESCRIPTION
**Pull request by chris-short**
Let the comments and commits begin! 🧡🧡🧡

Signed-off-by: Chris Short <chris@chrisshort.net>

**Which issue(s) this PR fixes**:

Fixes #4437

### CODE REVIEW

**Comment by chris-short** [comment]
/assign @parispittman 
/assign @chris-short 
/assign @onlydole 
/assign @kbarnard10 
/assign @jonasrosland 
/assign @simplytunde 

I don't know the others' handles so please assign as you see fit.

File: communication/marketing-team/CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+-leads make calls on smaller editorial and transactional (example: twitter)
+
+TODO
+
+### Off-Ramp
+
+Things change, priorities get optimized, life happens. Having an easy on and off ramp is important to this team. If a lead needs to take an extended absence or leave the team, a shadow would be offered the lead position prior to seeking a lead elsewhere. Therefore moving from shadow to lead and vice versa should be made as simply as possible by continued communications from this team about its members, roles, successes, and failures.
+
+(quit/leave/boot)
```
Comment (chris-short) [inline communication/marketing-team/CHARTER.md]: We need to look at what other teams are doing in this area and if we want to adopt it now or later.

File: communication/marketing-team/CHARTER.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+* Social Media
+* Web/Blog
+
+All roles should have at least one shadow. All leads should actively recruit shadows for the entire team.
+
+TODO
+
+### Decision Making
+
+-lazy consensus (policy)
```
Comment (chris-short) [inline communication/marketing-team/CHARTER.md]: There's a team somewhere that has a lazy consensus thing in their charter. If it is logical for this group then copy and pasting it is fine, in my opinion.

File: communication/marketing-team/CHARTER.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+-leads make calls on smaller editorial and transactional (example: twitter)
```
Comment (chris-short) [inline communication/marketing-team/CHARTER.md]: Leads are encouraged to seek guidance from @kubernetes/sig-contributor-experience and/or other community members/teams as appropriate.

**Comment by parispittman** [comment]
thanks @chris-short!

**Comment by parispittman** [comment]
area/contributor-comms

**Pull request merged**